dp-frontend-homepage-controller.nomad

```diff
@@ -0,0 +1,147 @@
+job "dp-frontend-homepage-controller" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "web"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "15s"
+      interval = "1m"
+      mode     = "delay"
+    }
+
+    task "dp-frontend-homepage-controller-web" {
+      driver = "docker"
+
+      artifact {
+        source = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-frontend-homepage-controller/{{REVISION}}.tar.gz"
+      }
+
+      config {
+        command = "${NOMAD_TASK_DIR}/start-task"
+
+        args = ["./dp-frontend-homepage-controller"]
+
+        image = "{{ECR_URL}}:concourse-{{REVISION}}"
+
+        port_map {
+          http = 8080
+        }
```

Comment:
Don't need port_map. These are only for Java service. These three lines can be removed.

Reply:
👍

Reply:
:+1